pkg/cmd/issue/list/http.go

```diff
@@ -149,10 +149,17 @@ func searchIssues(client *api.Client, repo ghrepo.Interface, filters prShared.Fi
 
 	perPage := min(limit, 100)
 
+	var searchType string
+	if advancedSearch {
+		searchType = "ISSUE_ADVANCED"
+	} else {
+		searchType = "ISSUE"
+	}
+
```

Comment:
Where were you thinking that feature detection would determine if advanced search is available?

https://github.com/cli/cli/blob/12ff91fc8fa9dfd03f8beec4f14f6a9c998efdf6/internal/featuredetection/feature_detection.go#L65-L95

One example where `IssueFeatures` is used:

https://github.com/cli/cli/blob/12ff91fc8fa9dfd03f8beec4f14f6a9c998efdf6/pkg/cmd/issue/list/list.go#L144-L151